### DISCUSSION TITLE
Question on if stats include commits made with different email

### DISCUSSION THREAD

**Discussion by wena04** [anuraghazra/github-readme-stats#3777] [Q&A] [answered]
My account has 2 emails, and for some projects I use different emails and Name as for the commit. Would the statistics take into account that and show the results of both? No matter which email or Name I use to make the commit it will still be counted into the stats right?

**Comment by juan-miii** [comment]
If the emails you use are linked to your account (check Emails in Settings) you should be good. Any email in there including the one generated by GitHub should count as they are linked to your account.

**Comment by qwerty541** [comment, marked as answer]
Hey @wena04, I see only top languages card inside your GitHub profile readme, so I assume that the question is exactly about it's behavior. This card shows language usage only inside your own non-forked repositories, not depending on who the author of the commits is. It does not include your contributions into another users/organizations repositories. Therefore, there is no difference in how many emails your are using, it does not affect on this card.

**Reply by wena04** [reply]
How would I change it so that it can show the contributions into another users/organizations repositories?

**Reply by qwerty541** [reply]
As far as I know, currently there are no way to get this data from GitHub API. If you want this behavior to be improved you can support [this feature request](https://github.com/orgs/community/discussions/18230) created by [@rickstaa](https://github.com/rickstaa) inside GitHub Community.

**Reply by juan-miii** [reply]
Agree with @qwerty541, info is related only to code on your own repositories, and about emails, as I mentioned before, it is related to the account, not the mail of the commit.
